pkg/controller/statusmanager/status_manager.go

```diff
@@ -165,17 +219,18 @@ func (status *StatusManager) set(reachedAvailableLevel bool, conditions ...confi
 		if err != nil {
 			buf = []byte(fmt.Sprintf("(failed to convert to YAML: %s)", err))
 		}
+
 		if isNotFound {
 			if err := status.client.Create(context.TODO(), co); err != nil {
 				return err
 			}
-			log.Printf("Created ClusterOperator with conditions:\n%s", string(buf))
+			log.Printf("Set ClusterOperator conditions:\n%s", buf)
```

Comment:
I know this was already present in the code, but I see a mix in CNO of klog and log, maybe it's worth switching to just one in other PR.

Reply:
yeah, we should switch to klog but I'm avoiding mixing in a given package.